inc/config.php

```diff
@@ -45,6 +45,25 @@
 Session::setPath();
 Session::start();
 
+//init cache
+try {
+   $cache_conf = GLPI_CACHE_CONFIG;
+   if (!isset($cache_conf['adapter']['options'])
+       || !isset($cache_conf['adapter']['options']['namespace'])) {
+      $cache_conf['options']['namespace'] = 'glpicache';
```

Comment:
is this a typo ? (missing [adapter] level)

Reply:
No, the goal is to force namespace when not specified in GLPI_CACHE_CONFIG (see define.php)

Reply:
but you test isset($cache_conf['adapter']['options']['namespace']) and then set $cache_conf['options']['namespace'] , this looks inconsistent

Reply:
Indeed, you are right! Sorry :)

Reply:
Fixed